app/src/main/java/com/framgia/moviedbtraining/movieDetails/MovieDetailsActivity.kt

```diff
@@ -52,6 +72,75 @@ class MovieDetailsActivity : AppCompatActivity(), MovieDetailsContract.ViewModel
   }
 
   override fun showMoviePosters(moviePosters: List<MovieDetails.Posters>) {
-//Todo fetch all posters
+    val mAdapter: PostersAdapter = PostersAdapter(moviePosters, R.layout.item_posters)
+    rvMovieImages.layoutManager = LinearLayoutManager(this, LinearLayoutManager.HORIZONTAL, true)
+    rvMovieImages.adapter = mAdapter
+  }
+
+  fun setMovie() {
+    tvName.text = mMovie.title
+    initValue(RELEASE_DATE)
+    initValue(BUDGET)
+    initValue(REVENUE)
+    initValue(RUNTIME)
+    initValue(OVERVIEW)
+    initValue(RELEASE_STATUS)
+    initValue(GENRE)
+    initValue(RATING)

```

Comment:
@nambingo Because the values needed to be formatted before set values. Also need to reUse and clean